components/bl-audio-recorder-component/src/index.js

```diff
@@ -0,0 +1,153 @@
+import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
+
+import { captureMediaDevices, download } from './helpers';
+
+const { cn, normalizeDimensionValue } = BackendlessUI.CSSUtils;
+
+export default function AudioRecorder({ component, eventHandlers, elRef }) {
+  const { player, controls, noise, fileName, fileType, width, display, style, classList } = component;
+  const { onStart, onStop, onDownload, onState } = eventHandlers;
+  const { paused, recording, inactive } = STREAM_STATES;
```

Comment:
It could be done out of a component as it constant, otherwise it runs every render